graph mode: round out relu support

vkuzo opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack):
* #37771 graph mode: more in-place activation handling
* **#37592 graph mode: round out relu support**

Summary:

Makes sure that all the standalone relu flavors are tested in
graph mode.

Test Plan:

```
python test/test_quantization.py TestQuantizeScriptPTSQOps.test_swap_dequantize
```

Reviewers:

Subscribers:

Tasks:

Tags:

Differential Revision: [D21366597](https://our.internmc.facebook.com/intern/diff/D21366597)

vkuzo committed:
graph mode: round out relu support

Summary:

Makes sure that all the standalone relu flavors are tested in
graph mode.

Test Plan:

```
python test/test_quantization.py TestQuantizeScriptPTSQOps.test_swap_dequantize
```

Reviewers:

Subscribers:

Tasks:

Tags:

[ghstack-poisoned]

jerryzh168 reviewed (approved):
Thanks!

jerryzh168 commented on test/quantization/test_quantize_script.py:
```diff
@@ -1673,6 +1676,12 @@ def forward(self, x):
                 x = F.elu(x)
                 x = self.hardsigmoid(x)
                 x = F.hardsigmoid(x)
+                x = self.relu(x)
```
btw, please add inplace versions as well

vkuzo committed:
Update on "graph mode: round out relu support"


Summary:

Makes sure that all the standalone relu flavors are tested in
graph mode.

Test Plan:

```
python test/test_quantization.py TestQuantizeScriptPTSQOps.test_swap_dequantize
```

Reviewers:

Subscribers:

Tasks:

Tags:

[ghstack-poisoned]

vkuzo committed:
Update on "graph mode: round out relu support"


Summary:

Makes sure that all the standalone relu flavors are tested in
graph mode.

Test Plan:

```
python test/test_quantization.py TestQuantizeScriptPTSQOps.test_swap_dequantize
```

Reviewers:

Subscribers:

Tasks:

Tags:

[ghstack-poisoned]

vkuzo committed:
Update on "graph mode: round out relu support"


Summary:

Makes sure that all the standalone relu flavors are tested in
graph mode.

Test Plan:

```
python test/test_quantization.py TestQuantizeScriptPTSQOps.test_swap_dequantize
```

Reviewers:

Subscribers:

Tasks:

Tags:

Differential Revision: [D21366597](https://our.internmc.facebook.com/intern/diff/D21366597)

[ghstack-poisoned]

vkuzo committed:
Update on "graph mode: round out relu support"


Summary:

Makes sure that all the standalone relu flavors are tested in
graph mode.

Test Plan:

```
python test/test_quantization.py TestQuantizeScriptPTSQOps.test_swap_dequantize
```

Reviewers:

Subscribers:

Tasks:

Tags:

Differential Revision: [D21366597](https://our.internmc.facebook.com/intern/diff/D21366597)

[ghstack-poisoned]
